examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py

```diff
@@ -258,12 +264,6 @@ def evaluate_model_on_worker(
 
 
 if __name__ == "__main__":
-    # Logging setup
-    FORMAT = "%(asctime)s | %(message)s"
-    logging.basicConfig(format=FORMAT)
-    logger = logging.getLogger("run_websocket_client")
-    logger.setLevel(level=logging.DEBUG)
-
```

Comment:
Is this not needed?

Reply:
I moved at the top of the file - but rerun the examples and it seems it is ok how it is now.